src/nms-api/NMSConnectionFactory.cs

```diff
@@ -1,440 +1,460 @@
-/*
```

Comment:
I'm not sure what happened here, but the whole file seems to be recreated.

Reply:
not sure either, cant seem to fix it either. contents are same, just extra methods for context